pkg/sdk/custom_shapes.go

```diff
@@ -41,6 +41,10 @@ type customShapeInjector struct {
 // into the list of shapes in the API and update the list of custom shapes in
 // the SDKAPI object.
 func (h *Helper) InjectCustomShapes(sdkapi *ackmodel.SDKAPI) error {
+	err := h.InjectSimpleSchemaShapes(sdkapi)
+	if err != nil {
+		return err
+	}
```

Comment:
Should we make `InjectSimpleSchemaShapes` part of `customShapeInjector` ? feels like that's where it should be living. :)